packages/hoppscotch-common/src/components/app/Banner.vue

```diff
@@ -0,0 +1,50 @@
+<template>
+  <div
+    class="flex items-center px-4 py-2 text-tiny"
+    :class="bannerColor"
+    :role="bannerRole"
+  >
+    <icon-lucide-info
+      v-if="banner.type === 'info'"
+      class="mr-2 text-secondaryDark"
+    />
+    <icon-lucide-alert-circle
+      v-else-if="banner.type === 'warning'"
+      class="mr-2 text-secondaryDark"
+    />
+    <icon-lucide-alert-triangle v-else class="mr-2 text-secondaryDark" />
```

Comment:
We could simplify this by having an object with a map of banner types and the component references, and we can dynamically render the icon, [Ref](https://vuejs.org/guide/essentials/component-basics.html#dynamic-components).